admin/resources/js/components/PoolTable.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useMemo } from "react";
+import { defineMessages, useIntl } from "react-intl";
+import commonMessages from "./commonMessages";
+import { GetPoolsQuery, useGetPoolsQuery } from "../api/generated";
+import { navigate, useLocation } from "../helpers/router";
+import { notEmpty } from "../helpers/util";
+import { FromArray } from "../types/utilityTypes";
+import Button from "./H2Components/Button";
+import Table, { ColumnsOf } from "./Table";
+import Pill from "./H2Components/Pill";
+
+const messages = defineMessages({
+  columnUniqueIdentifier: {
+    id: "PoolTable.column.uniqueIdentifier",
+    defaultMessage: "Id",
+    description: "Title displayed on the Pool table Unique Identifier column.",
+  },
+  columnOwnerEmail: {
+    id: "PoolTable.column.email",
+    defaultMessage: "Email",
+    description: "Title displayed for the Pool table email column.",
+  },
+  columnGroupAndLevel: {
+    id: "PoolTable.column.groupAndLevel",
+    defaultMessage: "Group and Level",
+    description: "Title displayed for the Pool table Group and Level column.",
+  },
+  columnEditTitle: {
+    id: "userTable.column.editTitle",
+    defaultMessage: "Edit",
+    description: "Title displayed for the User table Edit column.",
+  },
+});
+
+type Data = NonNullable<FromArray<GetPoolsQuery["pools"]>>;
+
+export const PoolTable: React.FC<GetPoolsQuery & { editUrlRoot: string }> = ({
+  pools,
+  editUrlRoot,
+}) => {
+  const intl = useIntl();
+  const columns = useMemo<ColumnsOf<Data>>(
+    () => [
+      {
+        Header: intl.formatMessage(messages.columnUniqueIdentifier),
+        accessor: "id",
+      },
+      {
+        Header: intl.formatMessage(messages.columnOwnerEmail),
+        accessor: ({ owner }) => owner?.email,
+      },
+      {
+        Header: intl.formatMessage(messages.columnGroupAndLevel),
+        accessor: ({ classifications }) =>
+          classifications?.map((classification) => {
+            let counter = 0;
+            counter += 1;
+            return (
+              <Pill
+                key={counter}
```

Comment:
Instead of creating a new variable for counter before this, couldn't you use the following value for the key?
```suggestion
                key= {`${classification?.group}-${classification?.level}`}
```

```

Reply:
Oh yeah, that is way better. I just stubbed that in quickly and meant to revisit. Good catch!